gtsam_unstable/nonlinear/tests/testConcurrentBatchFilter.cpp

```diff
@@ -17,7 +17,7 @@
  */
 
 #include <gtsam_unstable/nonlinear/ConcurrentBatchFilter.h>
-#include <gtsam/slam/PriorFactor.h>
+#include <gtsam/nonlinear/PriorFactor.h>
```

Comment:
Remove?

Reply:
This file initializes multiple PriorFactors and reuses them in a single test.